code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -160,9 +179,279 @@
+	var/location = get_turf(holder.my_atom)
+	var/datum/effect/effect/system/chem_smoke_spread/S = new /datum/effect/effect/system/chem_smoke_spread
+	S.attach(location)
+	playsound(location, 'sound/effects/smoke.ogg', 50, 1, -3)
+	spawn(0)
+		if(S)
+			S.set_up(holder, 10, 0, location)
+			S.start()
+			sleep(10)
+			S.start()
+		if(holder && holder.my_atom)
+			holder.clear_reagents()
+	return
+
+/datum/reagent/sonic_powder
+	name = "Sonic Powder"
+	id = "sonic_powder"
+	description = "Makes a deafening noise."
+	reagent_state = LIQUID
+	color = "#000000"  //rgb: 96, 165, 132
+
+/datum/chemical_reaction/sonic_powder
+	name = "sonic_powder"
+	id = "sonic_powder"
+	result = "sonic_powder"
+	required_reagents = list("oxygen" = 1, "cola" = 1, "phosphorus" = 1)
+	result_amount = 3
+
+
+/datum/chemical_reaction/sonic_powder_deafen
+	name = "sonic_powder_deafen"
+	id = "sonic_powder_deafen"
+	result = null
+	required_reagents = list("sonic_powder" = 1)
+	required_temp = 374
+
+/datum/chemical_reaction/sonic_powder_deafen/on_reaction(var/datum/reagents/holder, var/created_volume)
+	var/location = get_turf(holder.my_atom)
+	playsound(location, 'sound/effects/bang.ogg', 25, 1)
+	for(var/mob/living/carbon/C in get_hearers_in_view(5, location))
+		if(ishuman(C))
+			var/mob/living/carbon/human/H = C
+			if((H.ears && (H.ears.flags & EARBANGPROTECT)) || (H.head && (H.head.flags & HEADBANGPROTECT)))
+				continue
+		C.show_message("<span class='warning'>BANG</span>", 2)
+		C.Stun(5)
+		C.Weaken(5)
+		C.setEarDamage(C.ear_damage + rand(0, 5), max(C.ear_deaf,15))
+		if(C.ear_damage >= 15)
+			C << "<span class='warning'>Your ears start to ring badly!</span>"
+		else if(C.ear_damage >= 5)
+			C << "<span class='warning'>Your ears start to ring!</span>"
+
+/datum/chemical_reaction/sonic_powder/on_reaction(var/datum/reagents/holder, var/created_volume)
+	if(holder.has_reagent("stabilizing_agent"))
+		return
+	holder.remove_reagent("sonic_powder", created_volume)
+	var/location = get_turf(holder.my_atom)
+	playsound(location, 'sound/effects/bang.ogg', 25, 1)
+	for(var/mob/living/carbon/C in get_hearers_in_view(5, location))
+		if(ishuman(C))
+			var/mob/living/carbon/human/H = C
+			if((H.ears && (H.ears.flags & EARBANGPROTECT)) || (H.head && (H.head.flags & HEADBANGPROTECT)))
+				continue
+		C.show_message("<span class='warning'>BANG</span>", 2)
+		C.Stun(5)
+		C.Weaken(5)
+		C.setEarDamage(C.ear_damage + rand(0, 5), max(C.ear_deaf,15))
+		if(C.ear_damage >= 15)
+			C << "<span class='warning'>Your ears start to ring badly!</span>"
+		else if(C.ear_damage >= 5)
+			C << "<span class='warning'>Your ears start to ring!</span>"
+
+/datum/reagent/phlogiston
+	name = "Phlogiston"
+	id = "phlogiston"
+	description = "+1 BURN, +1 BURNING and sets you alight. Don't be deceived by that meager +1, having phlogiston in the bloodstream can be very dangerous without a fire extinguisher or firebot. Even then, they can't prevent you from spontaneously igniting every cycle."
```

Comment:
Recommended description: "Sets the target on fire, getting worse and worse, and causes them to keep burning as long as it's in the bloodstream"